MMUL-BASH-01/Exercises/Day-3/Exercise.001.solutions

```diff
@@ -0,0 +1,34 @@
+1) Use the ifconfig [interface] command:
+
+   [mmul@localhost ~]$ ifconfig eth0
+   
+2) grep into the file:
+   
+   [mmul@localhost ~]$  sudo grep -i dns /etc/sysconfig/network-scripts/ifcfg-eth0
+   DNS1=192.168.100.3
+   DNS2=192.168.100.7
+   
+3) Use nc
+
+   [mmul@localhost ~]$ nc -v 192.168.100.7 22
+   Ncat: Version 7.50 ( https://nmap.org/ncat )
+   Ncat: Connection refused.
+   
+   Not reachable (22 is the SSH port - In this case, it's unreachable because
+   the destination host is a Windows host, SSH is not configured)
+
+4) With netstat
```

Comment:
The point is missing the answer to the original question, so is ssh accessible or not?